data/newtabicons-content.js

```diff
@@ -82,6 +84,7 @@ function updateThumbnails() {
         thumb.style.backgroundImage = thumb.dataset.newPreview;
         thumb.removeEventListener('mouseover', mouseOverListener);
         thumb.removeEventListener('mouseout', mouseOutListener);
+        window.addEventListener('mousemove', mouseMoveListener);
```

Comment:
Oh, yeah, it doesn't match up because the toggle one is describing the _next_ state.
And I agree it only needs the listener for case 2, so we should have "window.removeEventListener" in the other two cases, right?